trusted-types/support/csp-violations.js

```diff
@@ -39,10 +39,12 @@ function trusted_type_violations_and_exception_for(fn) {
     }
     // Force a connect-src violation. WebKit additionally throws a SecurityError
     // so ignore that. See https://bugs.webkit.org/show_bug.cgi?id=286744
+    // Firefox throws a NS_ERROR_CONTENT_BLOCKED exception, so ignore that too.
+    // See https://bugzilla.mozilla.org/show_bug.cgi?id=1951698
     try {
-      new EventSource("/common/blank.html");
```

Comment:
EventSource doesn't seem to trigger a SecurityPolicyViolation event for workers in Firefox. So I've switched to WebSocket which does throw.

Reply:
Worth double checking this hasn't broken other reporting tests cc @fred-wang

Reply:
wpt-firefox-nightly-results will probably tell us that, although I don't know if they are run with the flag on.